haiku-apps/trax/trax-1.1.1.recipe

```diff
@@ -0,0 +1,59 @@
+SUMMARY="Find files by name AND location, unlike Haiku's Find"
+DESCRIPTION="TraX is not a \"query\" tool. It does not keep a database of files and attributes. \
+It is a traditional find tool, which searches the disk every time, looking for files that meet \
+your criteria. The find-criteria are not Haiku specific, and it does not (currently) search \
+attributes.
+
+TraX comes in two varieties.
+
+ * FastTraX is similar to the Windows find tool. Windows users should find its features very \
+familiar.
+ * TraX is a cross between the UNIX \"find\" command the the MacOS \"Find Files\" tool. It has \
+much of the power of \"find\" with a Mac-like interface (a Mac on steroids that is)."
+HOMEPAGE="https://github.com/HaikuArchives/TraX/"
+COPYRIGHT="1999-2000, Tim Vernum"
+LICENSE="GNU GPL v2"
+REVISION="1"
+srcGitRev="ed09bd45e6cd9faa365c8c1f542f1636aa19cd63"
+SOURCE_URI="https://github.com/HaikuArchives/TraX/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="8a4261a787fa861ee40b78448e73e00e50800d2b660945c813a14ca5df31c8b4"
+SOURCE_DIR="TraX-$srcGitRev"
+
+ARCHITECTURES="all"
+
+PROVIDES="
+	trax = $portVersion
+	app:FastTraX = $portVersion
+	app:TraX = $portVersion
+	lib:libfind = $portVersion
+	lib:libTraX = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc
+	cmd:make
+	"
+
+BUILD()
+{
+	make
+}
+
+INSTALL()
+{
+	# make install doesn't really do anything, it is done by the default target already
+	# but keep it here if this is fixed later.
+	make install
+
+	mkdir -p $appsDir $libDir
+	cp -a bin/FastTraX bin/TraX $appsDir
+	cp -a bin/lib/*.so $libDir
```

Comment:
Is installing the libraries to `$libDir` correct? Upstream installs them in a `lib` subdirectory, but I think having them in `$libDir` is nicer than in a subdirectory of `$appsDir`. Or should everything be put in a subdirectory of `$appsDir`, given there are two versions of the app?

Reply:
> Upstream installs them in a `lib` subdirectory

A lot of old BeOS apps did that so they could be distributed as a zip file.  You could just extract and run the app right away because `%A/lib` is in `$LIBRARY_PATH`.  This isn't necessary when installing from a package, so I always move them to `$libDir`.